edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/requests/IRequestManager.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Core.Requests
+{
+    using System.Threading.Tasks;
+
+    public interface IRequestManager
+    {
+        Task<(int, string)> ProcessRequest(string request, string payloadJson);
```

Comment:
Is it possible to create a data object for this return instead of the tuple. That way we can document the purpose of the fields.

Reply:
I added variables to the types.. that should convey the intent.